examples/fft_physical_x/fft_r2c_x.f90

```diff
@@ -141,12 +141,20 @@ program fft_r2c_x
    end do
    !$acc end loop
 
-   call MPI_ALLREDUCE(error, err_all, 1, real_type, MPI_SUM, MPI_COMM_WORLD, ierror)
-   err_all = err_all / (real(nx, mytype) * real(ny, mytype) * real(nz, mytype))
+   call MPI_ALLREDUCE(MPI_IN_PLACE, error, 1, real_type, MPI_SUM, MPI_COMM_WORLD, ierror)
+   error = error / (real(nx, mytype) * real(ny, mytype) * real(nz, mytype))
+
+   ! Abort if the error is too high
```

Comment:
This line is duplicated

Reply:
Fixed in https://github.com/2decomp-fft/2decomp-fft/pull/310/commits/6ce600ec82bde39b9a62ade92899ea06dfa639ce